creator-node/src/routes/prometheusMetricsExporter.js

```diff
@@ -0,0 +1,33 @@
+const Prometheus = require('prom-client')
+
+const {
+  handleResponse,
+  sendResponse,
+  errorResponseServerError
+} = require('../apiHelpers')
+const PrometheusMetric = require('../services/PrometheusMetric')
+
+const metricPrefix = 'audius_cn_'
+
+// collect default metrics recommended by Prometheus:
+// event loop lag, active handles, GC, and Node.js version
+Prometheus.collectDefaultMetrics({ prefix: metricPrefix })
```

Comment:
While this file, `prometheusMetricsExporter.js`, replies on `prom-client`, it strictly houses the route that exports the collected metrics.

`PrometheusMetric.js` is a wrapper around the `prom-client` library that collects metrics for the "exporter" above.

`Prometheus.collectDefaultMetrics()` should only be called once and should occur before the first route is requested. I figured since `metricPrefix` was also required at this time and `collectDefaultMetrics()` was tied closer to the exporter and not our collected metrics, that it should appear next to the "exporter".